sdk-docs/publish-app/marketplace-config.md

```diff
@@ -81,3 +81,5 @@ marketplace: {
     tags: [],
 },
 ```
+
+We recommend studying the iris-app-manifest.ts file before building your app, as it will give you an intuition of the constraints that your app can be built under.
```

Comment:
Just a small suggestions, might make it a bit simpler "as it will give you an understanding of the constraints within which your app can be built."